### PR TITLE
Add agenda items for Nov 28

### PR DESCRIPTION
**Pull request by binji**
Copied from https://github.com/WebAssembly/meetings/pull/123.

### CODE REVIEW

**Comment by binji** [comment]
Gonna add some other conversations that folks have brought up too

File: 2017/CG-11-28.md [hunk 1]
```diff
@@ -32,6 +32,32 @@ The meeting will be a Google Hangouts call.
...
+       * See [WebAssembly/threads/issues/49](https://github.com/WebAssembly/threads/issues/49),
+         discussion stalled
+       * Poll-worthy?
+    1. The type of immutable globals exported from instances
+       * See [WebAssembly/threads/issues/73](https://github.com/WebAssembly/threads/issues/73),
+         discussion stalled
+       * Basically, either change existing API incompatibly or live with complex API
+       * Poll-worthy?
+    1. The type of the wake count argument and return value for atomic.wake
+       * Spec text changed this to i64 but poll at CG mildly in favor of older i32
```
Comment (jfbastien) [inline 2017/CG-11-28.md]: Can you quote the actual poll results here?

File: 2017/CG-11-28.md [hunk 2]
```diff
@@ -32,6 +32,32 @@ The meeting will be a Google Hangouts call.
...
+       * See [WebAssembly/threads/pull/79](https://github.com/WebAssembly/threads/pull/79)
+       * POLL: Should we limit these instructions to only being valid during instantiation?
+    1. Returning `WebAssembly.Global` object instead of Number for exported globals
+       * See [WebAssembly/threads/issues/73](https://github.com/WebAssembly/threads/issues/73)
+       * POLL: Do we want to have a uniform return value for exported immutable/mutable globals?
+       * POLL: Should we specify that immutable globals return `WebAssembly.Global`?
+    1. Require custom sections after all known sections
+       * See [WebAssembly/design/issues/1153](https://github.com/WebAssembly/design/issues/1153)
+       * We currently allow custom sections anywhere, but since most (all?) producers currently put
+         them at the end, we may have incompatibilities in implementations.
```
Comment (jfbastien) [inline 2017/CG-11-28.md]: What are the incompatibilities? I don't see any such mention in the bug.

Comment (binji) [reply 2017/CG-11-28.md]: Luke was just speculating that it is likely undertested and likely to have problems. Should I omit?

Comment (jfbastien) [reply 2017/CG-11-28.md]: Yes.

Comment (binji) [reply 2017/CG-11-28.md]: OK, we didn't get to this item anyway.

**Pull request merged**